src/main.cpp

```diff
@@ -14,21 +14,23 @@
 static void printHelp(const char* app)
 {
     printf("Print BMC inventory list.\n");
-    printf("Copyright (c) 2020 YADRO.\n");
+    printf("Copyright (c) 2021 YADRO.\n");
     printf("Version " VERSION "\n");
     printf("Usage: %s [OPTION...]\n", app);
     printf("  -n, --name=NAME  Print item with specified name only\n");
     printf("  -a, --all        Also print non-present units\n");
     printf("  -e, --empty      Also print empty properties\n");
     printf("  -j, --json       Print in JSON format\n");
     printf("  -h, --help       Print this help and exit\n");
+    printf("  -H, --host       Get data from remote host over SSH\n");
```

Comment:
Why do we need this? To read inventory from another BMC?

Reply:
I guess that is to have a way to debug `lsinventory` on the developer station and retrieve data from a real BMC stand.
But I've agreed with Artem, it would be better to define a compile-time flag that is able to disable that option in the production.

Reply:
moved under `#ifdef` and configuration option